doc/nrf/working_with_nrf/nrf70/developing/regulatory_support.rst

```diff
@@ -0,0 +1,358 @@
+     - U-NII-4
+     -
+   * - 177
+     - 5875
+     - 5895
+     - 5885
+     - U-NII-4
+     -
+
+Conformance to regulatory restrictions
+**************************************
+
+Once a regulatory domain is configured, the following restrictions are imposed on the operation of the nRF70 Series devices:
+
+1. Channels: The device will operate only on the channels allowed in that country.
+#. Scan mode: The device will use Active scans where allowed and resort to Passive scans on channels where Active scans are not allowed.
+#. DFS: DFS is enabled on the channel.
+   The device will operate in the following two states:
+
+   * Scan state: The channel operation must be in the Passive mode.
+   * Connected state: The device will switch to other directed channel by an access point in case of radar detection.
+#. Transmit power: The maximum transmit power on each channel is restricted to a value allowed for that channel in that country.
+
+In the absence of a user configured country setting, the nRF70 Series device configures to the world regulatory domain by default.
+The AP beacon message may have the regulatory hints such as country code information, to be considered in the region configuration settings.
+
+
+Configuration options
+*********************
+
+You can configure the regulatory domain through one of the following options:
+
+Build time
+==========
+
+Use the :kconfig:option:`CONFIG_NRF700X_REG_DOMAIN` Kconfig option to set the regulatory region.
+The regulatory region will take a ISO/IEC alpha2 country code for the country in which the device is expected to operate.
+
+Run time
+========
+
+You can also set the regulatory domain using either an API or a shell command as shown in the following table:
+
+.. list-table:: Wi-Fi regulatory domain network management APIs
+   :header-rows: 1
+
+   * - Network management APIs
+     - Command
+     - Description
+     - Expected output
+   * - | net_mgmt(NET_REQUEST_WIFI_CMD_REG_DOMAIN, <XY>, force=true)
+       | (Set option 1)
+     - ``wifi reg_domain <XY> -f``
+     - Override country code regulatory hints
+     - Wi-Fi regulatory domain is set to: <XY>
+   * - | net_mgmt(NET_REQUEST_WIFI_CMD_REG_DOMAIN, <XY>)
+       | (Set option 2)
+     - ``wifi reg_domain <XY>``
+     - Set regulatory domain to <XY>
+     - Wi-Fi regulatory domain is set to: <XY>
+
+If you set a specific country or region with ``force=true`` flag (Set option 1), the beacon's country or region information will be ignored, and the current region will be overridden with the set value.
+
+In automatic region setting, the beacon's regulatory region (if present) will be given high precedence over the following scenarios:
+
+* The set region during booting and Kconfig options.
+* The user configured country region (``force=false`` flag (Set option 2)).
+
+Once the nRF70 Series device disconnects with the AP, the device restores the country code to world regulatory if it was connected to the beacon's country code.
```

Comment:
Try re-word more accurately. E.g. 
Once the nRF70 Series device disconnects from the AP, the device restores the country code to world regulatory if automatic mode was enabled.